File: Clients/Java/Delete.java
import org.json.simple.JSONObject;

public class Delete{
    public static void main(String[] args) {
        PhoenixDB pdb = new PhoenixDB("localhost", 9999, "sudhanshu", "Password@123", "college");

        JSONObject filter = new JSONObject();
        filter.put("name", "Amit");
        filter.put("surname", "Sharma");

        String response = pdb.delete("students", filter.toJSONString());
        System.out.println(response);

        pdb.close();
    }
}

File: Clients/PHP/delete.php
<?php

    print("PhoenixDB<br>");

    require_once('phoenixDB.php');
    $phoenixDB = new PhoenixDB("localhost", 9999, "sudhanshu", "Password@123", "college");
    sleep(1);

    $filter = json_encode(
        array(
            'name' => 'Sudhanshu',
            'surname' => 'Jha'
        )
    );
    $response = $phoenixDB->delete("students", $filter);
    print('<pre>');
    print_r($response);
    print('</pre>');

?>